libp2p/Network.h

```diff
@@ -17,6 +17,7 @@
 /** @file Network.h
  * @author Alex Leverington <nessence@gmail.com>
  * @author Gav Wood <i@gavwood.com>
+ * @author ijcoe6ru <s6yg1ez3@mail2tor.com>
```

Comment:
Yes, @ijcoe6ru.   We are only using author tags for substantial contributions (if at all).
Much easier to keep these globally.    Who has touched particular files is way too granular to be useful.